runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/catalog/CatalogHandler.java

```diff
@@ -18,6 +18,11 @@
 public interface CatalogHandler
 {
     int NO_SCHEMA_ID = 0;
+    String TEST = "test"; // Added for unit test & IT purpose
+    String SCHEMA_REGISTRY = "schema-registry";
+    String INLINE = "inline";
```

Comment:
These cannot go here, caused by abstraction leak and this is a public API for engine.